stock_quantity_history_location/wizards/stock_quantity_history.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_stock_quantity_history_location" model="ir.ui.view">
+        <field name="name">Inventory Report at Date and Location</field>
```

Comment:
```suggestion
        <field name="name">stock.quantity.history.form - stock_quantity_history_location</field>
```
Perhaps we can take the chance to update the view name to make it more searchable